spec/acceptance/generic_events/runner_spec.rb

```diff
@@ -310,14 +310,11 @@
     end
 
     context 'when event_name=lockout' do
-      # NB: lockout events have should have no effect on a move, they are purely for auditing
       before { create(:event_move_lockout, eventable: move) }
 
-      it 'does not update the move status' do
```

Comment:
I think it's worth keeping this test as we still wouldn't want the status of a move to change after this event.